framework/include/fvbcs/FVFluxBC.h

```diff
@@ -0,0 +1,35 @@
+//* This file is part of the MOOSE framework
```

Comment:
Could FV bcs live in the same place as FE BCs?

Reply:
How are DG bcs handled?  FV Dirichlet bcs are fundamentally different than other BCs and have basically zero overlap with how they need to be handled compared to FE BCs - bc value used to initialize ghost elements used for running flux kernels.  Also, all FV bcs have/require an FV variable member variable (hah!) rather than an FE variable member variable.  We already have separate blocks for DG kernels and FV kernels too.  I'm not sure how we would combine them since they share no common c++ ancestry.  Can multiple classes just registerBase the same string to show up as the same syntax?  Does that cause any other problems?

Reply:
DG uses `IntegratedBCs` even for "Dirichlet" BCs. Example: https://github.com/idaholab/moose/blob/next/framework/src/bcs/DGFunctionDiffusionDirichletBC.C

Reply:
Ghost element BCs are unique to FV, flux BCs are very similar to Neumann in concept, but when you need them in the threaded loop you'd have to treat them separately anyway.

Reply:
That DG BC looks like a weak BC where you need to run all the kernel physics using the dirichlet value - i.e. all the stuff we managed to do automagically for users using ghost elements.  We should look into updating DG to using a face loop with ghost bc stuff too (future work).  @permcody - what would it take to stick a FVBCs (which share no c++ heritage with BCs) in the same input file BCs block?